### PR TITLE
e2e/verify: refactor to support hypershift

### PR DESCRIPTION
**Pull request by jbpratt**
- e2e/verify: enable imagestreams test on hypershift
- e2e/verify: enable PVCs test on hp
- e2e/verify: run SCC tests on hp
- e2e/verify/fips: refactor and skip on hypershift
- e2e/verify: cleanup and run pods test on hypershift
- e2e/verify: refactor and simplify loadbalancers test
- e2e/webhooks: move and simplify prevents workloads test
- e2e/verify: consolidate and simplify annotations tests

### CODE REVIEW

File: pkg/e2e/verify/sccs.go [hunk 1]
```diff
@@ -25,243 +28,86 @@ func init() {
...
-		isSccRule := false
-		for _, resource := range rule.Resources {
-			if resource == "securitycontextconstraints" {
-				isSccRule = true
+	ginkgo.DescribeTable("should include", func(ctx context.Context, scc string) {
+		clusterRole := &rbacv1.ClusterRole{}
+		expect.NoError(client.Get(ctx, "dedicated-admins-cluster", "", clusterRole))
+		for _, rule := range clusterRole.Rules {
+			if slice.ContainsString(rule.Resources, "securitycontextconstraints", nil) && slice.ContainsString(rule.Verbs, "use", nil) {
+				Expect(slice.ContainsString(rule.ResourceNames, scc, nil)).To(BeTrue())
```
Comment (jbpratt) [inline pkg/e2e/verify/sccs.go:44]: TODO: need to add explanation to this error

**Comment by ryankwilliams** [comment]
For [SDCICD-902](https://issues.redhat.com/browse/SDCICD-902)

**Pull request merged**